src/ctia/properties.clj

```diff
@@ -60,7 +60,7 @@
 
 (s/defschema StorePropertiesSchema
   "All entity store properties for every implementation"
-  (let [configurable-stores (map name (keys @store/stores))
+  (let [configurable-stores (map name (keys store/empty-stores))
```

Comment:
Preferring `empty-stores` at top-level makes it possible to move `store/stores` into a TK service.